lib/model/KnowledgePanelElement.dart

```diff
@@ -46,7 +50,29 @@ class KnowledgePanelTextElement extends JsonObject {
       unknownEnumValue: KnowledgePanelTextElementType.DEFAULT)
   final KnowledgePanelTextElementType? type;
 
-  const KnowledgePanelTextElement({required this.html, this.type});
+  /// Human readable source language (eg: "English")
+  @JsonKey(name: 'source_language')
+  final String? sourceLanguage;
+
+  /// Source locale name (eg: "en")
+  @JsonKey(name: 'source_lc')
+  final String? sourceLocale;
+
+  /// Name of the source (eg: "Wikipedia")
+  @JsonKey(name: 'source_text')
+  final String? sourceText;
+
+  /// Link to the source (eg: "https://en.wikipedia.org/wiki/Sodium acetate")
+  @JsonKey(name: 'source_url')
+  final String? sourceUrl;
+
+  const KnowledgePanelTextElement(
+      {required this.html,
+      this.type,
+      this.sourceLanguage,
+      this.sourceLocale,
+      this.sourceText,
+      this.sourceUrl});
```

Comment:
A trailing comma + reformat would be nice.

Reply:
This is not the case for all constructors in this file (eg: KnowledgePanelTableCell, KnowledgePanelPanelIdElement…).
Should I add only to this particular class or to the whole file?

Reply:
Just this one. The less I need to read, the better.